packages/syft/src/syft/core/node/common/action/run_class_method_action.py

```diff
@@ -222,6 +237,18 @@ def execute_action(self, node: AbstractNode, verify_key: VerifyKey) -> None:
                 resolved_self.read_permissions = result_read_permissions
                 resolved_self.write_permissions = result_write_permissions
         if not isinstance(result, StorableObject):
+            # TODO: Upload object to seaweed store, instead of storing in redis
+            # create a proxy object class and store it here.
+            if RunClassMethodAction.check_send_to_blob_storage(
+                settings=node.settings, obj=result
+            ):
+                result = upload_result_to_s3(
+                    asset_name=self.id_at_location.no_dash,
```

Comment:
MAYBE: could we rename it to asset_id, so we could distinction with dataset_name and id?